doc/source/cluster/configure-manage-dashboard.md

```diff
@@ -0,0 +1,304 @@
+      middlewares = ["test-redirectregex", "strip"]
+      service = "dashboard"
+  [http.middlewares]
+    [http.middlewares.test-redirectregex.redirectRegex]
+      regex = "^(.*)/ray/dashboard$"
+      replacement = "${1}/ray/dashboard/"
+    [http.middlewares.strip.stripPrefix]
+      prefixes = ["/ray/dashboard"]
+  [http.services]
+    [http.services.dashboard.loadBalancer]
+      [[http.services.dashboard.loadBalancer.servers]]
+        url = "http://localhost:8265"
+```
+
+## Disabling the Dashboard
+
+Dashboard is included if you use `ray[default]`, `ray[air]` or {ref}`other installation commands <installation>` and automatically started.
+
+To disable the dashboard, use the following arguments `--include-dashboard`.
+
+::::{tab-set}
+
+:::{tab-item} Single-node local cluster
+
+**Start the cluster explicitly with CLI** <br/>
+
+```bash
+ray start --include-dashboard=False
+```
+
+**Start the cluster implicitly with `ray.init`** <br/>
+
+```{testcode}
+:hide:
+
+import ray
+ray.shutdown()
+```
+
+```{testcode}
+import ray
+ray.init(include_dashboard=False)
+```
+
+:::
+
+:::{tab-item} VM Cluster Launcher
+To disable the dashboard while using the {ref}`VM cluster launcher <vm-cluster-quick-start>`, include the `ray start --head --include-dashboard=False` argument
+in the `head_start_ray_commands` section of the [cluster launcher's YAML file](https://github.com/ray-project/ray/blob/0574620d454952556fa1befc7694353d68c72049/python/ray/autoscaler/aws/example-full.yaml#L172).
+:::
+
+:::{tab-item} Kuberay
+To be added
+:::
+::::
+
+
+## Viewing Built-in Dashboard API Metrics
+
+The dashboard is powered by a server that serves both the UI code and the data about the cluster via API endpoints.
+There are basic Prometheus metrics that are emitted for each of these API endpoints:
+
+`ray_dashboard_api_requests_count_requests_total`: Collects the total count of requests. This is tagged by endpoint, method, and http_status.
+
+`ray_dashboard_api_requests_duration_seconds_bucket`: Collects the duration of requests. This is tagged by endpoint and method.
+
+For example, you can view the p95 duration of all requests with this query:
+
+```text
+
+histogram_quantile(0.95, sum(rate(ray_dashboard_api_requests_duration_seconds_bucket[5m])) by (le))
+```
+
+These metrics can be queried via Prometheus or Grafana UI. Instructions on how to set these tools up can be found {ref}`here <observability-visualization-setup>`.
+
+
+(observability-visualization-setup)=
+## Integrating with Prometheus and Grafana
+Optional integration with Prometheus and Grafana enables 2 valuable features:
+- Default Grafana dashboard templates provided by Ray that visualize the important system metrics for monitoring and debugging.
+- View the time-series metrics in Ray Dashboard directly via embedded Grafana visualizations.
+
+
+For better Ray Dashboard experience, i.e., viewing time-series metrics, users need to set up Prometheus and Grafana and integrate them with Ray Dashboard.
```

Comment:
```suggestion
For the enhanced Ray Dashboard experience, like viewing time-series metrics, set up Prometheus and Grafana to integrate with Ray Dashboard.
```

Reply:
Should we provide a link to the instructions?

Reply:
Yeah. we should